src/api/goods.js

```diff
@@ -6,6 +6,13 @@ export function getAll() {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => fetch(API_URL)
+  .then(response => response.json())
+  .then(goods => [...goods]
```

Comment:
You can use getAll function here and then choose first 5 out of it. No need to repeat the code